src/opnsense/scripts/unbound/start.sh

```diff
@@ -61,3 +63,7 @@ chown -R unbound:unbound /var/unbound
 
 /usr/local/sbin/unbound -c /var/unbound/unbound.conf
 /usr/local/opnsense/scripts/unbound/cache.sh load
+
+if [ ! -z "${DOMAIN}" ]; then
+  /usr/local/opnsense/scripts/dhcp/unbound_watcher.py --domain ${DOMAIN}
```

Comment:
might want to check tabs vs spaces here, I "try" to use tabs for shell scripts and it looks mixed in here